azure/Guidelines.md

```diff
@@ -747,6 +749,20 @@ While removing a value from an enum is a breaking change, adding value to an enu
 
 > :ballot_box_with_check: **You SHOULD** use extensible enums unless you are positive that the symbol set will **NEVER** change over time.
 
+### Deprecating Behavior Notification
+
+When the [API Versioning](#API-Versioning) guidance above cannot be followed and the [Azure Breaking Change Reviewers](mailto:azbreakchangereview@microsoft.com) approve a breaking change to the operation/service, the operation/service that is being deprecated must add the `azure-deprecating` response header with a semicolon-delimited string notifying the caller what is being deprecated, when it will no longer function, and a URL linking to more information. The purpose is to inform customers (when debugging/logging responses) that they must take action to modify their call to the service's operation or their call will soon stop working entirely. It is not expected that client code will examine/parse this header's value in any way; it is purely informational to a human being.
```

Comment:
The current intro only mentions APIs that are doing breaking changes, however, I believe we should recommend/enforcing teams that have API deprecations in stable/preview APIs as well even if there is no breaking behavior.

For example - "This API version is going away by xyz", unless you see that as breaking of course, I figure "breaking" in this context means people change a given API version to act differently after a given date. So we might want to clarify this

Reply:
Azure's policy is that a given API version can NEVER change its behavior. So, the definition of "breaking" is that the next version **requires** that old clients update their code. When the new/breaking version ships is when all the previous versions are deprecated. This is when the header would be returned from customer calls specifying any previous version.